source/administration/changelog.md

```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Added support for showing the channel name in the message box.
+ - Added support for markdown in plugin system console help text fields.
+ - Added ability to convert Excel cells to markdown table when pasting in Mattermost.
+ - Added ability to render emojis in interactive message buttons.
+ 
+#### Plugins
+ - Added plugin support for bot accounts.
+ - Created a plugin component to override file previews.
+ - Added support for plugins to create link tooltips.
+ 
+#### Bulk Import/Export
+ - Added User Preference fields in bulk export.
+ - Added ability to include direct and group message channels and their posts in bulk export.
+ - Added ability to include deactivated users in bulk import.
+ 
+#### CLI
+ - Created CLI command ``command show`` to allow seeing detailed information of a slash command.
+ - Created CLI command ``webhook show`` to allow seeing detailed information of a webhook.
+ - Created CLI command ``team rename`` to allow renaming teams.
+ - Created CLI command ``channel search`` to allow searching for channels.
+ 
+#### Administration
+ - Improved default session timeout behavour, including changing the default ``SessionLengthWebInDays`` from 30 to 180 days.
+ - Added full text search to the system console panel to easily find options in the configuration.
+ - (Advanced Permissions) Split managing emoji permissions into "create", "delete own" and "delete others".
+ - (Advanced Permissions) Added ``List_Public_Teams``, ``Join_Public_Teams``, ``List_Private_Teams`` and ``Join_Private_Teams`` permissions.
+ - Updated Mattermost to default to console logging in a human readable format.
+ - Added support for LDAP groups search.
+ - Added a setting to the system console to change the minimum length of hashtags.
+ - Added support for setting Reply-To header in outbound Mattermost emails.
+ - Added support for invalidating all email invitations from the system console.
+
+### Bug Fixes
+ - Fixed an issue where enterprise features became immediately unavailable when the enterprise license expired with a 15 day grace period.
+ - Fixed an issue where a channel did not get removed from the unreads section if the user navigated out of it via a permalink.
+ - Fixed an issue where a link from Access Control Groups to Group Filter on AD/LDAP did not work for subpath Site URL.
+ - Fixed an issue where expired channels appeared in "My Channels" section of channel switcher if using the **Automatically Close Direct Messages** setting.
+ - Fixed an issue where the text box reverted to default size after a user returned from the Integrations page.
+ - Fixed an issue where the profile popover wasn't allowed to close itself when opened through an at-mention.
+ - Fixed an issue where filtering by first name with Korean characters no longer worked for at-mentions.
+ 
+### Compatibility
+
+ - Deprecated configurable ``timezones.json`` in favour of the existing hard-coded list built into the server. **XXXX NEEDS MORE DETAILS**
```

Comment:
Is this a deprecated feature or just part of the Configuration in Database?

https://mattermost.atlassian.net/browse/MM-14260

Reply:
@lieut-data please comment here

Reply:
I think the description (minus the `XXXXX` part ;P)) is sufficient: it's independent of the configuration in the database changes.